lumicks/pylake/kymotracker/kymotrack.py

```diff
@@ -213,8 +215,25 @@ def create_track(time, coord, min_length=None):
             min_length = float(np.unique(min_length).squeeze())
         return KymoTrack(time.astype(int), coord, kymo, channel, min_length)
 
-    if (min_length_field := "minimum_length (-)") in data:
-        mandatory_fields.append(min_length_field)
+    if csv_version == 3:
+        warnings.warn(
+            RuntimeWarning(
+                "This CSV file is from a version in which the minimum observable track duration "
+                "was incorrectly rounded to an integer. This may result in errors in certain "
+                "analyses. To remedy this error, use `lk.filter_tracks()` on the "
+                "tracks from the widget with the desired minimum track length and resave "
+                "using the `KymoTrackGroup.save()` method. Note that the minimum length should be "
+                "greater or equal to the minimum length used for the original tracking. For more "
+                "information refer to "
+                "https://lumicks-pylake.readthedocs.io/en/latest/tutorial/nbwidgets.html#migrating-old-track-files "
```

Comment:
Feel free to ignore, but:
```suggestion
                "https://lumicks-pylake.readthedocs.io/en/latest/tutorial/nbwidgets.html#migrating-old-track-files"
```